### ISSUE TITLE
Site does not work properly

### ISSUE THREAD

**Issue by Dosojin1** [orangethewell/open-witness-library#1]
Hi brother,

This project looks great.
But I'm stumble on 1 problem. I couldn't not manage to let it work properly.

When I startup the project I only see this (see picture below), and when click on it, nothing happens.
I'm missing something? Or did I do something wrong?

Could you help me?

![image](https://github.com/orangethewell/open-witness-library/assets/1633112/758e2416-6854-4b28-847c-d872347548d5)

**Comment by Dosojin1** [comment]
I'm getting this error inside the console of the browser:
![image](https://github.com/orangethewell/open-witness-library/assets/1633112/2407405b-f0bf-4daf-87a7-518c4f6badb8)

**Comment by orangethewell** [comment]
Sorry for not adding any documentation about this! :sweat_smile: 

Basically, you need to add some publication on the directory specified by the tauri; Since I didn't managed to add multilanguage publications neven different types of publication currently, you will need to download Brazilian Portuguese publications and add them to the application data folder, located in your home directory.

On linux, it is:
`/home/{your user}/.local/share/open-witness-library/publications/T/bk/{the pub you downloaded}`
You will probably need to create the `publications/T/bk` path. Remembering that you need to **extract** the _.jwpub_ file there, and the `contents` to a folder named `content` inside the pub folder.

The final setup will look something like this:
![image](https://github.com/orangethewell/open-witness-library/assets/67232334/805eefce-65ab-462e-8946-6a52e31d9b20)

If it not works, wait a while. I'm targeting to add some option to add the publications soon :)

**Comment by Dosojin1** [comment]
Hi, Thanks for you answer!

Only one problem, I'm using windows 😂

**Comment by orangethewell** [comment]
It seems you need to [use `%appdata%` so](https://docs.rs/tauri/2.0.0-alpha.10/tauri/path/struct.PathResolver.html#method.data_dir).

I didn't get in knowledge if you installed tauri correctly, based on your error log.

If not, you need to follow tauri setup instructions:
```bash
cargo install tauri-cli # If you already installed cargo
cargo tauri dev # run this command on repo folder 
```

**Comment by deldesir** [comment]
Hi,

I am testing this on Linux.

I got this error running cargo tauri dev:
Error `tauri.conf.json` error on `tauri > security`: Additional properties are not allowed ('assetProtocol' was unexpected)

When running it with trunk serve, I only see update list.

Disclaimer: I copied some jwpub files from /home/{your user}/.local/share/open-witness-library/publications/T/bk/ following your instructions to only keep the manifest.json outside the content directory.

Thanks in advance for your guidance.

**Comment by Dosojin1** [comment]
@deldesir 
I could let it work with `cargo tauri dev` by removing some config in `tauri.conf.json`.
Navigate to `src-tauri\tauri.conf.json`
And replace there
```
"security": {
  "csp": null,
  "assetProtocol": {
    "enable": true,
    "scope": ["$DATA/**", "$HOME/**"]
  }
},
```
with
```
"security": {
  "csp": null
},
```

Then `cargo tauri dev` should work properly

**Comment by deldesir** [comment]
Thanks @Dosojin1. Now the build process is running. Will update once it finishes.

**Comment by deldesir** [comment]
The build passed. I applied some fixes along the way with `cargo fix`. All the publications are detected and I can see their titles in the terminal after clicking on update list, in the app window. The problem is once I click on "update list", the app crashes.

The error:
`thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src-tauri/src/jwpub/manager.rs:141:123`

Do you guys have an idea how to fix it?

**Comment by orangethewell** [comment]
Fixed with [v0.1.0 release](https://github.com/orangethewell/open-witness-library/releases/tag/0.1.0)
